dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/processor/queue/TaskResponseService.java

```diff
@@ -66,13 +66,32 @@
     @Autowired
     private ProcessService processService;
 
+    @Autowired
+    private MasterConfig masterConfig;
+
     /**
      * task response worker
      */
     private Thread taskResponseWorker;
 
+    /**
+     * event handler
+     */
+    private Thread eventHandler;
+
     private ConcurrentHashMap<Integer, WorkflowExecuteThread> processInstanceMapper;
 
+    private final ConcurrentHashMap<String, TaskResponsePersistThread> eventHandlerMap = new ConcurrentHashMap<>();
+
+    private ListeningExecutorService listeningExecutorService;
+
+    private ExecutorService eventExecService;
+
+    /**
+     * task response mapper
+     */
+    private final ConcurrentHashMap<Integer, TaskResponsePersistThread> processTaskResponseMapper = new ConcurrentHashMap<>();
```

Comment:
the name make me  think the map was a DAO object.

maybe change it to `processTaskResponseMap`